draft-ietf-rtcweb-jsep.xml

```diff
@@ -4076,6 +4076,65 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             process, as a remote offer or answer, unmodified SDP coming
             from any other implementation of this specification.</t>
         </section>
+        <section title="Processing RTP packets"
+                 anchor="sec.rtp.demux">
```

Comment:
It might help to have a bit of intro text here that helps people realize that a Receiver corresponds to a m-line and then there is a second stage of demux that gets it to the correct codec on the m-line.  I'm thinking of something along the lines of 

When using RTP without bundle, each m-line had a separate port that can be use to know which m-line an incoming RTP packets was associated with. With bundle there are additional identifiers that are used in getting the RTP packet to the correct m-line. This section describes how the information found in the SDP and the RTP packet are used to associated the packet with the correct m-line. Once this has happened, the same approaches and non bundle RTP are used to get the packet to the correct codec in that m-line. A RtpReceiver will correspond to a exactly one m-line in the SDP. An RtpReceiver can have multiple codecs associated with it each with their own RID and SSRC. The RID values are allocated by the receiver and send to the server in the SDP while the SSRC are allocated by the sender. Using RID to demultiplex allows the receiver to demultiplex incoming RTP using the RID values before the SDP answer is received.